backend/lcfs/web/api/notification/schema.py

```diff
@@ -31,9 +33,9 @@ class DeleteNotificationMessageResponseSchema(BaseSchema):
 class SubscriptionSchema(BaseSchema):
     notification_channel_subscription_id: Optional[int] = None
     is_enabled: Optional[bool] = True
-    notification_channel_id: Optional[int] = None
-    user_profile_id: Optional[int]= None
-    notification_type_id: Optional[int] = None
+    notification_channel_key: Optional[str] = None
```

Comment:
would it be nice to rename it to notification_channel_name, as "key" suffix is bit confusing.

Reply:
Good point! Renaming it to `notification_channel_name` makes sense. I also updated `notification_type_key` to `notification_type_name`. Thanks for the suggestion!